routes/v1.js

```diff
@@ -145,6 +145,7 @@ apiRoutes.get(
 
 apiRoutes.get(
   '/articles/:slug',
+  Auth.isLoggedIn,
```

Comment:
Let's do this in a way that if the user is logged in, you get the stats, but if the user isn't logged in, no stats. Or just anonymous readers.

Reply:
That's exactly what is happening in that block. The middleware sends the user details in `req.user` if they are logged in and returns `next()` if they're not.